Training_node/first_project/app/services/request.js

```diff
@@ -0,0 +1,16 @@
+const errors = require('../errors'),
```

Comment:
the service name should be something like albums, if you wanna do a wrapper for request its ok, but that goes in a helper